tests/test_hierarchy_helpers.hpp

```diff
@@ -309,8 +309,23 @@ class TestMFMeshEvaluator final
   {
   }
 
+  TestMFMeshEvaluator(
+      TestMFMeshEvaluator<dim, fe_degree, ScalarType> const &_other_evaluator)
+      : mfmg::DealIIMatrixFreeMeshEvaluator<dim>(*this),
+        _material_property(_other_evaluator._material_property),
+        _fe(_other_evaluator._fe),
+        _laplace_operator(_other_evaluator._laplace_operator)
+  {
+  }
```

Comment:
Why did you define the copy constructor?

Reply:
We call the copy constructor in `clone()` and I wanted to make sure we copy as little as possible here. In particular, we should need to copy `mutable` members since they are set up in `matrix_free_initialize_agglomerate` only.

Reply:
Please add a comment